pkg/onchain/dns/dns.go

```diff
@@ -10,16 +10,15 @@ import (
 	"github.com/massalabs/thyra/pkg/wallet"
 )
 
-const dnsRawAddress = "A1Q65NojVV5YPyZruVkeU1CGeS3tjLNwGSzAmZfAJPE5vuvus4C"
+const DnsRawAddress = "A12jkDPTcdhkqGg9VoKsTwvkBwZeSHQw7wJqQYKrNesKnjnGejuR"
```

Comment:
Why changing it to a public value ?

Reply:
Because I use it also in the `domain.go`
Like that I have only one reference in Thyra